server/src/Irony.cpp

```diff
@@ -245,7 +245,8 @@ void Irony::complete(const std::string &file,
     std::cout << "(\n";
 
     // re-use the same buffers to avoid unnecessary allocations
-    std::string typedtext, brief, resultType, prototype, postCompCar;
+    std::string typedtext, brief, resultType, prototype, postCompCar, access;
```

Comment:
I think it is actually exactly 80 characters so it should be ok.
I'm ok with available.